src/Http/BaseApplication.php

```diff
@@ -321,6 +345,11 @@ public function handle(
         $container->add(ServerRequest::class, $request);
         $container->add(ContainerInterface::class, $container);
 
+        /** @var \Cake\Event\EventManager $eventsManager */
+        $eventsManager = $this->events(EventManager::instance());
+        $this->pluginEvents($eventsManager);
+        EventManager::instance($eventsManager);
```

Comment:
This seems to be the best place to init the events, so that the DIC is already initialized and could technically be leveraged by users

Reply:
We should have a similar hook call for command line tools as well. It will be seen as a bug if events are not registered in commands.

Reply:
By similar call, I had intended to mean that we should also call `events` within `CommandRunner` after it bootstraps the application.